src/App.js

```diff
@@ -1,30 +1,20 @@
-// src/App.js
-import React from 'react';
-import {
-  BrowserRouter as Router, Routes, Route, Link,
-} from 'react-router-dom';
-import HomePage from './components/HomePage';
-import CategoriesPage from './components/CategoriesPage';
+import { Route, Routes } from 'react-router-dom';
+/* eslint-disable */
+import HomePage from './components/homePage';
+import CategoriesPage from './components/categoriesPage';
+/* eslint-enable */
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up: kindly remove this linter disabling. you should fix this as it complains about how you've imported those files but still, the linter disabling there kindly remove it.